### ISSUE TITLE
[bug]: Recent change to docker-compose-hub.yml is targeting non existent container label for plane-deploy

### ISSUE THREAD

**Issue by larkinwc** [makeplane/plane#2093]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

It appears that a recent change to the docker-compose-hub.yml included a new container stored at makeplane/plane-deploy. I double checked and it looks like the only label currently on docker hub for that container is develop. This breaks being able to deploy without changing the target label to 'develop'.

### Steps to reproduce

1. Checkout "master" (develop)
2. Follow self-hosting [guide](https://docs.plane.so/self-hosting) 
3. When running `docker compose -f docker-compose-hub.yml up` the following error response is received from docker `Error response from daemon: manifest for makeplane/plane-deploy:latest not found: manifest unknown: manifest unknown`

### Browser

Google Chrome

### Version

Self-hosted

**Comment by henit-chobisa** [comment]
Hey @larkinwc, thanks for sharing the issue, Let have a look and get back to you. ✌🏼

**Comment by Doldge** [comment]
from looking at dockerhub there is no `latest` tagged, only a `develop` per this URL https://hub.docker.com/r/makeplane/plane-deploy/tags

manually modifying the `docker-compose-hub.yml` file to point to instead will allow you to proceed, however I still ended up with a fairly large pile of errors in the docker logs and a non-functional instance of Plane.

specifically:
- `planedb` and `planebeatworker` both spent a period erroring while waiting on the `planebackend` to apply migrations [seems like the the beatworker should be delayed from running until the migrations have finished].
-  `planefrontend` erroring out on `MODULE_NOT_FOUND` 
- `planedeploy` also erroring out on a `MODULE_NOT_FOUND`

**Comment by nsauter** [comment]
Anything new here?

**Comment by JowaDev** [comment]
I found a temporary solution, but unfortunately, it doesn't allow us to benefit from the latest release. You need to clone the GitHub repository with the following command:

> git clone https://github.com/makeplane/plane.git plane

Then, switch to the commit of the previous release:

> sudo git checkout 6f397710ce3ecd75266f2349e5147a294e8f96d1

Afterward, you can follow the basic instructions from the README.md file, such as:

> chmod +x setup.sh
> sudo ./setup.sh http://localhost

Next, execute the Docker Compose command for the self-hosted version:

> sudo docker compose -f ./docker-compose-hub.yml up --build -d

Everything should work as expected...

**Comment by henit-chobisa** [comment]
Hey @larkinwc, @Doldge & @JowaDev, just wanted to let you know that we have resolved the above issue and have also introduced some new improvements, to make the self hosted experience, super smooth. The problem that you were facing above is because of the images not updating on dockerhub, which will be updated on our next release ( a few days ). I have mentioned the PRs that introduces the changes, you can have a look. 
Again, thank you so much for your time. 🙏🏽

https://github.com/makeplane/plane/pull/2133
https://github.com/makeplane/plane/pull/2115
https://github.com/makeplane/plane/pull/2135

**Comment by hifnStudio** [comment]
This is gonna take time so it's good to stick with v0.11. If anyone updated to newest version and getting error then change config for 0.11

https://github.com/makeplane/plane/issues/2151#issuecomment-1715622102

You can copy provided config to go live with older version.

**Comment by larkinwc** [comment]
> This is gonna take time so it's good to stick with v0.11. If anyone updated to newest version and getting error then change config for 0.11
> 
> [#2151 (comment)](https://github.com/makeplane/plane/issues/2151#issuecomment-1715622102)
> 
> You can copy provided config to go live with older version.

Yes, there should be a separate issue filed to ensure that versioned releases use image tags, since it looks like they currently do not.

**Comment by vihar** [comment]
We've implemented significant changes to enhance the stability of the self-hosted version. To address this issue, upgrade to the latest version. View documentation at https://docs.plane.so/self-hosting/. If you're still facing the issue, please let us know.

**Comment by grijalva10** [comment]
> We've implemented significant changes to enhance the stability of the self-hosted version. To address this issue, upgrade to the latest version. View documentation at https://docs.plane.so/self-hosting/. If you're still facing the issue, please let us know.

this url does not work

**Comment by mguptahub** [comment]
try this url https://docs.plane.so/docker-compose
